pkg/vsphere/extraconfig/encode.go

```diff
@@ -252,6 +253,9 @@ func encodeWithPrefix(src interface{}, prefix string) []types.BaseOptionValue {
 				switch field.Interface().(type) {
 				case time.Time:
 					config = append(config, &types.OptionValue{Key: key, Value: field.Interface().(time.Time).String()})
+				case net.IPNet:
+					n := field.Interface().(net.IPNet)
+					config = append(config, &types.OptionValue{Key: key, Value: n.String()})
```

Comment:
You can't do that for every type, but yes IPNet is a flat struct but it's not working with extraconfig for some reason. That's that's why I also created https://github.com/vmware/vic/issues/602 while creating this PR :)